api/mailer/SceGoogleApiHandler.js

```diff
@@ -147,6 +148,37 @@ class SceGoogleApiHandler {
       }
     });
   }
+
+  /**
+   * Grabs all the events for a given calendar by its id.
+   * will log the next 10 events into the console.
+   * @param calendarId {string} calendar id for which calendar to pull from
+   * @returns {{Array<Object>|Error)} The calendar events from Google or an
+   * error.
+   */
+  getEventsFromCalendar(calendarId, numOfEvents) {
+    // creates a const calendar for use in this method
```

Comment:
remove this comment, as this code is [self documenting](https://en.wikipedia.org/wiki/Self-documenting_code)